packages/help-center/src/components/help-center-container.tsx

```diff
@@ -48,38 +48,21 @@ const HelpCenterContainer: React.FC< Container > = ( {
 	}, [] );
 
 	const nodeRef = useRef< HTMLDivElement >( null );
-
 	const { setIsMinimized } = useDispatch( HELP_CENTER_STORE );
-	const [ isVisible, setIsVisible ] = useState( true );
 	const isMobile = useMobileBreakpoint();
 	const classNames = clsx( 'help-center__container', isMobile ? 'is-mobile' : 'is-desktop', {
 		'is-minimized': isMinimized,
 	} );
 
 	const onDismiss = useCallback( () => {
-		setIsVisible( false );
+		handleClose();
 		recordTracksEvent( `calypso_inlinehelp_close` );
-	}, [ setIsVisible ] );
-
-	const toggleVisible = () => {
-		if ( ! isVisible ) {
-			handleClose();
-			// after calling handleClose, reset the visibility state to default
-			setIsVisible( true );
-		}
-	};
+	}, [ handleClose ] );
 
 	const animationProps = {
```

Comment:
There isn't an animation anymore.

Reply:
@renancarvalho to consider for you when you're back.